projects/2020_07_25_Ongoing_Tech_Support.md

```diff
@@ -0,0 +1,139 @@
+
+### 3.2 Service Deliverables
+To achieve the goals above, we will provide on-duty technical support for future technical issues that the PKT Network will experience over the next 24 months. We can renew another Network Steward proposal and reach a new agreement after 24 months from the end of the current support period, if the proposal is accepted. 
+
+The technical support tasks include the following:
+
+* **1)** Collaborative debugging assistance within staff expertise. We plan to work as a team with the PKT Project to solve bugs.
+
+* **2)** System Administration Support (including monitoring and in-production observation)
+
+We will test and contribute to the existing PKT and related software: cjdns, PacketCrypt, PKT suite (including pktctl and pktd). Any new software created within the context of this Network Steward proposal will be created under the Artistic, MIT, or ISC license, Simplified 2-clause BSD License, or any other license in compliance with the Open Source Initiative (OSI) Open Source Definition after receiving explicit approval from the Network Steward, and it will be hosted on github.com/pkt-cash, if the network steward wishes it so. Any code contributions to existing licensed software within the PKT suite will be licensed in accordance with the licensing requirements of the software being modified.
+
+### 3.3 Execution Strategy
+Our execution strategy incorporates highly qualified personnel, and a responsive approach. We will take care of issues identified by Gridfinity's team or reported directly by an official member of the PKT Project or Network Steward board.
+
+
+
+## 4 Payments
+
+### 4.1 Price Breakdown
+
+* **Hours per month:** We estimate that our team currently spends an average of 20 hours per week working on troubleshooting the PKT network, and we project that the workload will continue to be the same for the next 24 months. The equivalent average hours per month is **80 hours / month**.
+
+* **Hourly rate:** Gridfinity's hourly rate is **PKT 25,000 / hour**, which includes the technical member salary, and other general overhead, such as Gridfinity's administrative tasks, equipment, supplies, and other expenses.
+
+* **PKT monthly payment:** 
+
+  80 hours/month x 25,000 PKT/hour = **2 million PKT/month**
+
+### 4.2 PKT Transfers
+Each payments shall be distributed to the following address: 
+
+`pkt1quem94ynpy6mnrgjn7jxc3rq7asp22a5at3fa93`
+
+Payments will be distributed to Gridfinity monthly, or, alternately, a different schedule negotiated with the Network Steward.
+
+
+
+## 5. Disclosure
+We hereby submit this application in good faith and we attest that we have made no effort, nor do we intend to make effort, influence the Network Steward to accept this or any other project we have submitted.
+
+1. Conflicts
+  1. [x] An organization is receiving the funds: *Gridfinity, LLC.*
+    1. [x] Organization has financial relationships with one or more reviewers: *Caleb James DeLisle* and *Adonis Gaitatzis*
```

Comment:
This relationship is true, but also Gridfinity has put in over a year of in-kind work into PKT and regardless of my relationship with Gridfinity or its members I believe their work speaks for itself in this regard.